openedx/core/djangoapps/user_api/accounts/image_helpers.py

```diff
@@ -88,7 +91,11 @@ def get_profile_image_urls_for_user(user):
 
     """
     if user.profile.has_profile_image:
-        return _get_profile_image_urls(_make_profile_image_name(user.username), get_profile_image_storage())
+        return _get_profile_image_urls(
+            _make_profile_image_name(user.username),
+            get_profile_image_storage(),
+            version=user.profile.profile_image_uploaded_at.strftime("%s"),
```

Comment:
This should probably be hashed, so we're not leaking information about when people updated their images.